### PR TITLE
fix: added animations in checkbox[123]

### PR DESCRIPTION
**Pull request by isha-dubey**
## Description

#1047 

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build

## Have you tested locally?

- [x] 👍 yes
- [ ] 🙅 no, because I am lazy

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md

### CODE REVIEW

File: src/packages/@app/styles/style.scss [hunk 1]
```diff
@@ -323,6 +323,7 @@ $theme-red-230408: #230408;
   --bg-secondary-1000 : #{$theme-grey-CCCCCC};
 
   --bg-tertiary-100 : #{$theme-violet-676A80};
+  --bg-tertiary-250 : #{$theme-violet-1C1D2B};
```
Comment (Astitva877) [inline src/packages/@app/styles/style.scss:326]: also add this style for border, icon etc.

File: src/packages/@app/styles/style.scss [hunk 2]
```diff
@@ -266,11 +266,12 @@ $theme-red-230408: #230408;
...
   --text-secondary-850 : #{$theme-grey-121212};
   --text-secondary-900 : #{$theme-grey-000000};
-  --text-secondary-1000 : #{$theme-grey-CCCCCC};
+  --text-secondary-1000: #{$theme-grey-CCCCCC};
 
   --text-tertiary-100 : #{$theme-violet-676A80};
   --text-tertiary-200 : #{$theme-violet-323548};
   --text-tertiary-250 : #{$theme-violet-1C1D2B};
+  --text-tertiary-350 : #{$theme-violet-1C1D2B};
   --text-tertiary-300 : #{$theme-violet-2A2C3C};
```
Comment (itsmdasifraza) [inline src/packages/@app/styles/style.scss:275]: duplicate not required

File: src/packages/@library/ui/button/Button.svelte [hunk 3]
```diff
@@ -135,4 +144,13 @@
   }
   .custom-btn-transparent:hover {
   }
+  .custom-btn-teritiary{
+    color: var(--text-primary-300); 
+    border: 1px solid var(--border-primary-200) !important;
+  }
+  .custom-btn-teritiary:hover{
+    color: var(--white-color); 
+    background-color: var(--bg-primary-300 );
```
Comment (itsmdasifraza) [inline src/packages/@library/ui/button/Button.svelte:153]: color: var(--text

Comment (itsmdasifraza) [reply src/packages/@library/ui/button/Button.svelte:153]: class name tertiary that represents violet not matching with background property that uses color primary that represents blue

Comment (isha-dubey) [reply src/packages/@library/ui/button/Button.svelte:153]: okay looking into it.

File: src/packages/@workspaces/features/save-as-request/layout/SaveAsRequest.svelte [hunk 4]
```diff
@@ -841,7 +841,8 @@
         }}
         title={"+ Collection"}
         buttonClassProp={"btn mb-2"}
-        buttonStyleProp={"color: var(--text-primary-300); font-size: var(--base-text); border: 1px solid var(--border-primary-200);"}
+        type={"teritiary"}
+        buttonStyleProp={"font-size: var(--base-text);"}
       />
     {:else if path.length > 0 && path[path.length - 1].type === ItemType.COLLECTION}
       <Button
```
Comment (itsmdasifraza) [inline src/packages/@workspaces/features/save-as-request/layout/SaveAsRequest.svelte:848]: could you please make a new button component in save-as-request/components/button/Button.svelte
and revert your type `tertiary`  changes from @library/ui/button/Button.svelte

File: src/packages/@workspaces/features/save-as-request/layout/SaveAsRequest.svelte [hunk 5]
```diff
@@ -850,7 +851,8 @@
         }}
         title={"+ Folder"}
         buttonClassProp={"btn mb-2"}
-        buttonStyleProp={"color: var(--text-primary-300); font-size: var(--base-text); border: 1px solid var(--border-primary-200);"}
+        type={"teritiary"}
+        buttonStyleProp={"font-size: var(--base-text);"}
       />
     {/if}
```
Comment (itsmdasifraza) [inline src/packages/@workspaces/features/save-as-request/layout/SaveAsRequest.svelte:857]: same component will be used here

File: src/packages/@library/ui/button/Button.svelte [hunk 6]
```diff
@@ -135,4 +144,13 @@
...
+    border: 1px solid var(--border-primary-200) !important;
+  }
```
Comment (itsmdasifraza) [inline src/packages/@library/ui/button/Button.svelte:155]: revert

File: src/packages/@library/ui/button/Button.svelte [hunk 7]
```diff
@@ -19,6 +19,7 @@
     | "danger"
     | "violet"
     | "transparent"
+    | "teritiary"
```
Comment (itsmdasifraza) [inline src/packages/@library/ui/button/Button.svelte:22]: revert

File: src/packages/@library/ui/button/Button.svelte [hunk 8]
```diff
@@ -52,6 +54,9 @@
     case BtnType.ICON:
       btnClass = "sparrow-icon-btn";
       break;
+    case BtnType.TERITIARY:
+      btnClass = "custom-btn-teritiary";
+      break;
```
Comment (itsmdasifraza) [inline src/packages/@library/ui/button/Button.svelte:59]: revert

File: src/packages/@library/ui/button/Button.svelte [hunk 9]
```diff
@@ -29,6 +30,7 @@
     TRANSPARENT = "transparent",
     OTHER = "other",
     ICON = "icon",
+    TERITIARY = "teritiary",
```
Comment (itsmdasifraza) [inline src/packages/@library/ui/button/Button.svelte:33]: revert

**Pull request merged**